projects/perf-benchmark.md

```diff
@@ -0,0 +1,86 @@
+### Description
+
+As the adoption of OpenTelemetry grows and larger enterprises continue to deepen
+their usage of project components there are persistent and ongoing end user
+questions about the OpenTelemetry performance impact. End user performance
+varies due to the quirks of their environment but without a project performance
+standard and historical data record no one really knows if the numbers they're
+seeing are abnormal or expected. Additionally, there is no comprehensive
+documentation available on tuning project components or the performance
+trade-offs available to users which results in a reliance on vendor support.
+
+The project needs to be able to track the current state of our
+components while preventing any performance regressions when making new
+releases. Customers need to be able to get a general sense of potential
+OpenTelemetry performance impact and the certainty that OpenTelemetry takes
+performance and customer resources seriously. Performance tracking and
+quantification is a general need that should be addressed by a project wide
+effort and automated tooling that minimizes repo owner effort while providing
+valuable new data points for all project stakeholders.  
+
+### Project Board
+
+### SIG Charter
+
+[charter](https://docs.google.com/document/d/1W0hLqHrUNtS2yclORTNtFZ-lXDblMzUYrp1PCSyzJr8/edit?usp=sharing)
```

Comment:
Once finalized, adding it to the markdown here would be good so it's preserved in the repository.